Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,47 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params string[] args)
+        {
+            if(baseLogger == null)
+            {
+                throw new ArgumentNullException();
+            }
+
+            baseLogger.Log(LogLevel.Error, message);
```

Comment:
This is dropping the args without formatting them as part of the message. This is why your BaseLoggerMixinTests are failing.
```suggestion
            baseLogger.Log(LogLevel.Error, string.Format(message, args));
```